web/frontend/src/app.vue

```diff
@@ -636,20 +627,19 @@ const clearFrameInterval = () => {
   window.clearInterval(cameraFrameIntervalId);
 };
 
-const viewCameraFrame = (cameraName: string, time: string) => {
+const viewCameraFrame = (cameraName: string, time: number) => {
   clearFrameInterval();
-  const selectedInterval = selectedMap[time as keyof typeof selectedMap];
 
-  if (time === 'Live') {
+  // Live
+  if (time === -1) {
     return;
   }
 
-  if (time === 'Manual Refresh') {
-    viewFrame(cameraName);
-  } else {
+  viewFrame(cameraName);
```

Comment:
logic change here: the first frame is fetched immediately. this fixes a bug where it would take 30 sec (or however many you have selected) for the first frame to come in. this is often not apparent because we start in Live mode, but if one switches to "30 sec" interval, toggle the cam off, toggle the cam on, then the screen would be blank for 30 sec.